docs/user_manual/components.md

```diff
@@ -695,15 +695,116 @@ A `fault` has no steady state output.
+- The rated voltage at the `tap_side` must be greater than or equal to the rated voltage at the `control_side`.
+- The voltage band must be sufficiently large, i.e., it must be greater than the largest change in voltage due to a change in tap position. This means, that
+  $U_{\text{band}} \geq \frac{\text{tap_size}/u_\text{tap_side}}{\left(1 + \left(\left|\text{tap_min}\right| - \left|\text{tap_nom}\right|\right)\frac{\text{tap_size}}{\text{u_tap_side}}\right)^2}u_{\text{control_side}}$
+- The line drop compensation is small, in the sense that its product with the typical current through the transformer is much smaller (in absolute value) than the smallest change in voltage due to a change in tap position. This means, that
+  $\left|z_{\text{compensation}}\right|\left|I_{\text{node}}\right| \ll \frac{\text{tap_size}/u_\text{tap_side}}{\left(1 + \left(\left|\text{tap_max}\right| - \left|\text{tap_nom}\right|\right)\frac{\text{tap_size}}{\text{u_tap_side}}\right)^2}u_{\text{control_side}}$
+
+These requirements make sure no edge cases with undefined behavior are encountered. Typical real-world power grids already satisfy these requirements and they should therefore not cause any problems.
+
+#### Steady state output
+
+| name      | data type | unit | description          |
+| --------- | --------- | ---- | -------------------- |
+| `tap_pos` | `int8_t`  | -    | optimal tap position |
+
+#### Short circuit output
+
+A `transformer_tap_regulator` has no short circuit output.
+
+#### Electric Model
+
+The transformer tap regulator itself does not have a direct contribution to the grid state.
+Instead, it regulates the tap position of the regulated object until the voltage at the control side is in the specified voltage band:
+
+$$
+   \begin{eqnarray}
+      U_{\text{control}} \in \left[U_{\text{set}} - \frac{U_{\text{band}}}{2}, U_{\text{set}} + \frac{U_{\text{band}}}{2}\right]
+   \end{eqnarray}
+$$
+
+##### Line drop compensation
+
+The transformer tap regulator tries to regulate the voltage in a specified virtual location in the grid, according to the folowing model.
+
+```txt
+tap_side   control_side                        part of grid where voltage is to be regulated
+------^\oo -*---------------virtual_impedance----*
+      |    U_node, I_node         Z_comp         U_control
+      |                                          |
+     regulator <=================================/
+```
+
+The control voltage is the voltage at the node, compensated with the voltage drop corresponding to the specified line drop compensation.
+
+$$
+   \begin{eqnarray}
+      & Z_{\text{compensation}} = r_{\text{compensation}} + \mathrm{j} x_{\text{compensation}} \\
+      & U_{\text{control}} = \left|\underline{U}_{\text{node}} - \underline{I}_{\text{node}} \cdot \underline{Z}_{\text{compensation}}\right|
+   \end{eqnarray}
```

Comment:
`u_control_side_node + i_out_transformer * z`
A node does not have a current. 

or maybe as is `u_control_side_node - i_transformer * z` with a repeat refence about the direction of current. The direction part always confuses the user at first.

Reply:
whoops, you're right.. fixed

Reply:
fixed again cause of the sign conventions: it's canceling minus signs

`u_control_side_node - i_out_transformer * z = u_control_side_node + i_transformer * z`